### PR TITLE
cross-compile for scala3 support

### LINKED ISSUE #10: Scala 3 support
Hi, is a release planned for Scala 3 (3.5.0)? Thanks

### PR DESCRIPTION
**Pull request by escouvois**
I created a cross-compiled version for Scala 3 to resolve issue #10. It seems to work, I was able to use it in a Scala 3 project as well.

### CODE REVIEW

**Comment by alaz** [comment]
Lovely! Thank you.

I think it deserves a minor version bump.

File: version.sbt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 
-version in ThisBuild := "1.0.4-SNAPSHOT"
```
Comment (alaz) [inline version.sbt:2]: I think it deserves a minor version bump.

**Review by alaz — DISMISSED** [review]
Lovely! Thank you.

**Comment by alaz** [comment]
FYI, `test:compile` seems to be failing

**Comment by escouvois** [comment]
Sorry I forgot to run tests... I fixed it.

**Review by alaz — APPROVED** [review]

**Comment by alaz** [comment]
Unfortunately, I failed to publish to Sonatype :

```
[error] stack trace is suppressed; run last publishSigned for the full output
[error] (publishSigned) java.net.ProtocolException: Server redirected too many times (20)
```

It's been 5 years after all. I will try to fix it later.

**Comment by alaz** [comment]
Phew, I did it: https://repo1.maven.org/maven2/com/osinka/i18n/scala-i18n_3/

**Comment by escouvois** [comment]
Awesome, thank you!

**Pull request merged**